pkg/virtctl/templates/templates.go

```diff
@@ -58,3 +64,15 @@ func ExactArgs(nameOfCommand string, n int) cobra.PositionalArgs {
 		return nil
 	}
 }
+
+// WarnPaused prints warning message if VMI is paused
+func WarnPaused(virtCli kubecli.KubevirtClient, name string, namespace string) {
```

Comment:
I would also consider changing `name` to `vmiName` to be more clear

Reply:
Changed.